src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -128,6 +131,14 @@ public ObservableList<Person> getFilteredPersonList() {
         return filteredPersons;
     }
 
+    /**
+     * Returns a list of tags currently defined in CampusConnect
+     */
+    @Override
+    public ObservableList<Tag> getListOfCurrentTags() {
+        return this.currentlyDefinedTags;
```

Comment:
Think since you need to display the tags sorted, can sort the tags here before passing it to the TagListPanel